media/doc/3rd Party Files.txt

```diff
@@ -147,7 +147,7 @@ URL: http://alter.org.ua/soft/win/uni_ata/
 
 Title: Miniport driver for the Buslogic BT 958 SCSI Controller
 Path: drivers/storage/port/buslogic
-Used Version: v1.2.0.2 (2005-08-17)
+Used Version: 1.2.0.2 (2005-08-17)
```

Comment:
This is VMWare's numbering scheme, not ours.

Reply:
> VMWare's numbering scheme

Really? I don't know where you got that from.

See
https://git.reactos.org/?p=reactos.git;a=blob;f=drivers/storage/port/buslogic/BusLogic958.rc;hb=HEAD#l26

Reply:
https://git.reactos.org/?p=reactos.git;a=blob;f=drivers/storage/port/buslogic/BusLogic958.c;h=76b6d8b024f8fe3614d836372326a130202ee1fb;hb=HEAD#l75

Reply:
I would rather trust an active line in .rc than a comment in .c.
Also, 'v' looks like an abbreviation to me, not a numbering part.